test/srcs/AssertReply.cpp

```diff
@@ -197,7 +197,15 @@ AssertReply& AssertReply::matches_entirely(const std::string& message)
     }
     return *this;
 }
-
+void AssertReply::print_target_and_messages(const std::string& target) {
+	std::cout << "Looking for: "<< target << std::endl << "in:" << std::endl;
+    for (std::vector<Message>::iterator it = _messages.begin(); it != _messages.end(); ++it) {
+		for (std::vector<std::string>::iterator itstring = it->args.begin(); itstring != it->args.end(); itstring++) {
+			std::cout << *itstring;
+		}
+		std::cout << std::endl;
```

Comment:
we should ensure it is not left in tests or use logger debug
```suggestion
		for (std::vector<std::string>::iterator itstring = it->args.begin(); itstring != it->args.end(); itstring++) {
			LOG_D_TEST("arg", *itstring);
		}
```